diff --git a/dist/index.mjs b/dist/index.mjs
index afd01449c0d03356760009a1efcd80c52f2c9fdf..0e4bda9f895e178f01f8b01d02eee53db201ea88 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -122,6 +122,9 @@ async function getChangelogEntry(cwd, release, releases, changesets, changelogFu
 	});
 	const relevantChangesets = changesets.filter((cs) => relevantChangesetIds.has(cs.id));
 	changelogLines.patch.push(Promise.resolve(changelogFuncs.getDependencyReleaseLine(relevantChangesets, dependentReleases, changelogOpts)));
+	if (typeof changelogFuncs.getChangelogEntry === "function") {
+		return changelogFuncs.getChangelogEntry(release, changelogLines);
+	}
 	const resolvedChangelogLines = {
 		major: await Promise.all(changelogLines.major),
 		minor: await Promise.all(changelogLines.minor),
